tests/Integration/Behaviour/Features/Scenario/Product/Combination/search_combinations_for_association.feature

```diff
@@ -69,6 +69,7 @@ Feature: Search combinations to associate them in the BO
       | ean13     | 978020137962      |
       | mpn       | mpn1              |
       | reference | ref1              |
+    # See DetailsAssertionFeatureContext::transformDetails
```

Comment:
you should instead put this `@see` tag inside php part of related step. That one allows you to navigate directly into that transform method and also you won't need to repeat it in 100 places